test/metabase/models/card_test.clj

```diff
@@ -775,3 +776,22 @@
                         Dashboard
                         before
                         changes)))))))))
+
+(deftest storing-metabase-version
+  (testing "Newly created Card should know a Metabase version used to create it"
+    (let [new-version "just a check"]
+      (t2.with-temp/with-temp [:model/Card card {}]
+        (is (= config/mb-version-string (:metabase_version card)))
+
+        (create-card-revision! (:id card) false)
+
+        (is (= config/mb-version-string
+               (-> (t2/select-one :model/Revision :model "Card" :model_id (:id card))
+                   :object :metabase_version)))
+
+        (with-redefs [config/mb-version-string new-version]
+          (t2/update! :model/Card :id (:id card) {:description "test"}))
+
+        (is (= new-version
+               (-> (t2/select-one :model/Card :id (:id card))
+                   :metabase_version)))))))
```

Comment:
style nit: In this case you are selecting a top-level key of a toucan model. There is a shorthand that is quite nice with toucan, that is very common to read in our codebase:
```
(t2/select-one-fn :metabase_version :model/Card :id (:id card))
```

Reply:
niiiice!